CHANGELOG.md

```diff
@@ -1,3 +1,18 @@
+## 2.32.0-rc.0 / 2021-12-01
+
+This list of changes is relative to [v2.32.0-beta.0](https://github.com/prometheus/prometheus/releases/tag/v2.32.0-beta.0), which was a special 2.32 beta release to pre-publish the Prometheus Agent features.
+
+* [FEATURE] TSDB: Add support for forwarding exemplars in Agent mode. #9664
+* [FEATURE] UI: Adapt web UI for Prometheus Agent mode. #9851
+* [ENHANCEMENT] Linode SD: Tune API request page sizes. #9779
+* [ENHANCEMENT] TSDB: Check postings bytes length when decoding. #9766
```

Comment:
This is not user facing

Reply:
Yeah, I wasn't sure about this one, but reading it again it doesn't seem like it's adding validation that would fix current user-visible bugs, etc., will remove.